.gitignore

```diff
@@ -126,6 +126,7 @@ shellAppWorkspaces
 /client-builds
 
 # Expo web env
+.env
```

Comment:
Let’s add the .env file now so we can also fix the mobile issue with the api. I was looking at the the expo docs earlier, and they do recommend using dotenv like we spoke about before. 

https://docs.expo.dev/guides/environment-variables/#storing-environment-variables-in-envrc-or-env

So let’s setup a .env file and replace all of the instances of ‘localhost’ when we call the apis with a ‘DOMAIN’ environment variable. 

There is a react example in the dotenv GitHub repo you can use for reference 

https://github.com/dotenv-org/examples/tree/master/dotenv-react